### DISCUSSION TITLE
Cannot install alevin-fry on Mac

### DISCUSSION THREAD

**Discussion by mircomacchi** [COMBINE-lab/alevin-fry#152] [Q&A] [answered]
Hello, I cannot install alevin-fry on my Mac laptop.
When running:

`conda install -c bioconda alevin-fry`

the error I encounter is the following :

```
Channels:
 - bioconda
 - defaults
 - conda-forge
Platform: osx-arm64
Collecting package metadata (repodata.json): done
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - alevin-fry

Current channels:

  - https://conda.anaconda.org/bioconda
  - defaults
  - https://conda.anaconda.org/conda-forge

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.
```

Any help would be appreciated.

**Comment by rob-p** [comment, marked as answer]
Hi @mircomacchi,

  It looks like while this was built for arm64 on Linux, it wasn't for Mac. I've fixed the recipe upstream. You can give it a try again.

Best,
Rob

**Comment by mircomacchi** [comment]
Great! It worked, many thanks!

I chose to try alevin-fry because it enables me to run quantifications directly on my laptop, which is sometimes more convenient than using a cluster. 
Also, the fact that it is open source is a big plus, as it allows users to raise questions and issues and get responses like in this case!
